### ISSUE TITLE
Unabe to install open-vm-tools 12.1.0

### ISSUE THREAD

**Issue by tvr2** [vmware/open-vm-tools#610]
### Describe the bug

Trying to upgrade open-vm-tools to v12.1.0
Current open-vm-tools version is v11.0.5
Operating system: Red Hat  7.9

When executing the command  './configure && make && make install' the following error occurs:
error: mspack >= 0.0.20040308alpha is required

How can i solve this?

### Reproduction steps

```bash
1.  ./configure && make && make install
```

### Expected behavior

v12.1.0 needs to be installed

**Comment by jonathanvmw** [comment]
@tvr2

Does your system have both the libmspack and libmspack-devel packages installed?

**Comment by johnwvmw** [comment]
@tvr2    I am assuming the your desire to build and install open-vm-tools 12.1.0 is to address CVE-2022-31676.  I have just left a meeting with Red Hat and have been told that security fixes are prepared and being reviewed for all versions of open-vm-tools that Red Hat currently supports.

Open-vm-tools 11.0.5 is the supported (by Red Hat) OVT version for RHEL 7.9.   Watch the errata notices for the fix's availability.

If you desire to continue with an OVT 12.1.0 build, be forewarned that:

1. there are many additional development and OSS packages that will be needed to do the OVT build.
2. the "make install" places the binaries, runtime and scripts in the "usual places" but does not contain the systemd/init.d start scripts that Red Hat packages.   The build does not result in RPM packages.  

Here is a more complete list of additional packges that you may need add to your RHEL 7.9 system in order to complete the build.

yum -y install \\
  make \\
  rpm-build \\
  autoconf \\
  automake \\
  libtool \\
  gcc-c++ \\
  doxygen \\
  fuse-devel \\
  glib2-devel \\
  gtk2-devel \\
  gtkmm24-devel \\
  gtkmm30-devel \\
  gtk3-devel \\
  libdnet-devel \\
  libicu-devel \\
  libmspack-devel \\
  libX11-devel \\
  libXext-devel \\
  libXi-devel \\
  libXinerama-devel \\
  libXrandr-devel \\
  libXrender-devel \\
  libXtst-devel \\
  openssl-devel \\
  pam-devel \\
  procps-devel \\
  libtool-ltdl-devel \\
  xmlsec1-devel \\
  xmlsec1-openssl-devel \\
  valgrind-devel \\
  libdrm-devel \\
  systemd-devel \\
  gawk \\
  curl

**Comment by johnwvmw** [comment]
Yes, Red Hat will announce updates and security fixes via errata notices.    

The "trick" is that you must opt into receiving the notices when you register the system for access to software repositories.   The notices are "off" be default.

Do a web search for something like "red hat how to get errata notices".
